spec/meilisearch/index/settings_spec.rb

```diff
@@ -52,6 +54,7 @@
       expect(settings['pagination'].transform_keys(&:to_sym)).to eq(default_pagination)
       expect(settings['filterableAttributes']).to eq([])
       expect(settings['sortableAttributes']).to eq([])
+      expect(settings['proximityPrecision']).to eq(default_proximityPrecision)
```

Comment:
`default_proximityPrecision` should be snake_cased to `default_proximity_precision`

Reply:
Changed